src/loaders/gtdb/gtdb_genome_attribs_loader.py

```diff
@@ -107,56 +101,63 @@ def _row_to_doc(row, kbase_collection, load_version):
     """
     Transforms row (from a dataframe) into ArangoDB collection document
     """
-    doc = {
-        FLD_ARANGO_KEY: md5_string(f"{kbase_collection}_{load_version}_{row.name}"),
-        FLD_COLLECTION_ID: kbase_collection,
-        FLD_LOAD_VERSION: load_version,
-    }
+    # parse genome id
+    genome_id = loader_helper.parse_genome_id(row.accession)
+    doc = loader_helper.init_genome_atrri_doc(kbase_collection, load_version, genome_id)
+
+    doc[FLD_COLLECTION_ID] = kbase_collection
+    doc[FLD_LOAD_VERSION] = load_version
+
     doc.update(row.to_dict())
 
     return doc
 
 
-def _df_to_docs(df, kbase_collection, load_version, header_mapper):
-    _rename_col(df, header_mapper)
```

Comment:
Can the _rename_col function be deleted?

Reply:
👍

Reply:
:+1: